packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -100,7 +104,10 @@ export function specTestIterator(
   opts: SkipOpts = defaultSkipOpts
 ): void {
   for (const forkStr of readdirSyncSpec(configDirpath)) {
-    if (opts?.skippedForks?.includes(forkStr)) {
+    if (
+      opts?.skippedForks?.includes(forkStr) ||
+      (process.env.SPEC_FILTER_FORK && forkStr !== process.env.SPEC_FILTER_FORK)
```

Comment:
Curious why we would want to specify fork that we want to skip in a env variable `SPEC_FILTER_FORK`. What's the use case for this? I think `defaultSkipOpts` is serving quite well

Reply:
Quickly run the spec tests for some forks. Whilde debugging, if we run for all forks it take too much time. 

```ts
SPEC_FILTER_FORK=electra yarn test:spec:minimal
```